src/blas/backends/cublas/cublas_scope_handle.hpp

```diff
@@ -28,12 +28,23 @@
 #include <sycl/backend/cuda.hpp>
 #endif
 #include <sycl/context.hpp>
-#include <sycl/detail/pi.hpp>
 #else
 #include <CL/sycl/backend/cuda.hpp>
 #include <CL/sycl/context.hpp>
+#endif
+
+// After Plugin Interface removal in DPC++ ur.hpp is the new include
+#if __has_include(<sycl/detail/ur.hpp>)
+#include <sycl/detail/ur.hpp>
```

Comment:
Please note that instead of removing a footgun you have just reloaded it so it will fire at you once again in upcoming months (or years) when we drop UR in favor of something else.

You should *not* use implementation details like this in your project. Why aren't use using the official SYCL interop interface, i.e. `backend_return_t<your_backend, context>`?

I'm also surprised that you are changing the type of that variable without changing any of its uses, meaning that the type doesn't matter for you because it will be erased at some point. Wouldn't it be better to use some opaque type right away to avoid dependency on particular SYCL implementation?